charts/vela-core/templates/defwithtemplate/metrics.yaml

```diff
@@ -1,3 +1,8 @@
+apiVersion: v1
```

Comment:
This will be overridden by `hack/vela-templates/definitions/metrics.yaml`. Please update there and run
```
./hack/vela-templates/gen_definitions.sh
```